<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFilePushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="regionLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnLineEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDataPushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateMetricsPushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Metrics</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minLineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxLineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="medianLineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Region</string>
    </property>
   </widget>
   <widget class="QLabel" name="columnLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Column</string>
    </property>
   </widget>
   <widget class="QLabel" name="minLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QLabel" name="medianLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>600</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Median</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorLineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
